activerecord/lib/active_record/associations.rb

```diff
@@ -1251,6 +1273,14 @@ module ClassMethods
         #   persisted new records placed at the end.
         #   When set to +:nested_attributes_order+, the index is based on the record order received by
         #   nested attributes setter, when accepts_nested_attributes_for is used.
+        # [:before_add]
+        #   Defines an {association callback}(https://guides.rubyonrails.org/api/classes/ActiveRecord/Associations/ClassMethods#module-ActiveRecord::Associations::ClassMethods-label-Association+callbacks) that gets triggered before an object is added to the association collection.
```

Comment:
```suggestion
        #   Defines an association callback[rdoc-ref:Associations::ClassMethods@Association+callbacks] that gets triggered before an object is added to the association collection.
```

I think that will work, I'm also curious what would work for markdown /cc @skipkayhil